offer-manager-domain/src/main/groovy/life/qbic/business/offers/content/CreateOfferContent.groovy

```diff
@@ -77,7 +77,7 @@ class CreateOfferContent implements CreateOfferContentInput, FetchOfferOutput{
         groupProductItems(offer.items)
         String id = offer.identifier.toString()
         def offerContentBuilder = new OfferContent.Builder(offer.customer,offer.selectedCustomerAffiliation,offer.projectManager,creationDate,expirationDate,offer.projectTitle
-        ,offer.projectObjective, offer.experimentalDesign.orElse(""),offer.identifier.toString())
+        ,offer.projectObjective, offer.experimentalDesign.orElse(""),offer.identifier.toString(), offer.getTotalNetPriceWithOverheads())
```

Comment:
we could add a builder constructor which obtains the offer object, so we dont need to adjust the constructor all the time 🤔

Reply:
We might have to change the builder after all. The current implementation does not provide compile-time checks and only produces runtime errors. So it might be better to have it like for all the other builders that the mandatory arguments are in the constructor and only optional arguments are supplied as fluent API. That is why I included it in the constructor atm

Reply:
I see, it was just a thought that I wanted to document. Do you mind document the request in an issue so we keep track of it?

Reply:
https://github.com/qbicsoftware/offer-manager-2-portlet/issues/758